libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -628,6 +644,15 @@ AP_GPS_UBLOX::_parse_gps(void)
                     break;
                 }
             }
+        } else if(_msg_id == MSG_ACK_NACK) {
+            switch(_buffer.ack.clsID) {
+            case CLASS_CFG:
+                switch(_buffer.ack.msgID) {
+                case MSG_CFG_SBAS:
+                    _unconfigured_messages &= ~CONFIG_SBAS;
```

Comment:
Why would we accept a NACK for SBAS config? If we want to enable/disable then we should be trying to make that happen?